waspc/packages/ts-inspect/test/exports.test.ts

```diff
@@ -1,34 +1,36 @@
 import * as path from 'path';
 import { getExportsOfFiles } from "../src/exports";
 
+// TODO(before merge): run these tests in CI
```

Comment:
Looking for help with this, I don't know what the best way to go about this is.

Reply:
Happy to help but give me a bit more info: which part are you not sure about, what have you tried?

I am not sure myself but I guess we would just add a step to our github actions yaml file that runs tests from this package. You can find that file at the top of our repo, in .github/workflows/ci.yaml . Or did you have some specific dillema?

Reply:
I was mostly hoping you/someone would already know what the correct way to do this is for npm packages. But I'll look it up and see what people say.